pkg/kn/commands/trigger/update_flag.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trigger
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// TriggerUpdateFlags are flags for create and update a Trigger
+type TriggerUpdateFlags struct {
+	Broker string
+	Filter []string
```

Comment:
We usually name array of things in the plurals. So this would be `Filters []string` not `Filter`

Reply:
fixed.